src/community/label_propagation.jl

```diff
@@ -1,24 +1,24 @@
 """
 Community detection using the label propagation algorithm (see [Raghavan et al.](http://arxiv.org/abs/0709.2938)).
-`g`: imput Graph
+`g`: input Graph
 `maxiter`: maximum number of iterations
 return : vertex assignments and the convergence history
 """
```

Comment:
also you want to add to your new function to `doc/build.jl`  in the community section, and the run
```julia
julia doc/build.jl
```
so it will show up in the documentation on readthedocs